templates/wishlist_add.html

```diff
@@ -13,9 +13,12 @@
                                <p class="form-title">Create new wish</p>
                                    <form class="wishlist" action="" method="post">
                                        {{ form.hidden_tag() }}
+                                       <label style="color:white;font-style:italic;font-size:12px;">{{form.type.label}}</label>
```

Comment:
I think it's better to follow _loose coupling_ principle and put this styling it in `css` file.

Reply:
ok, thank you 👍